builder/build.js

```diff
@@ -166,7 +166,12 @@ function copyModuleScripts() {
                 '(function(factory, root) {',
                 '    if (typeof define == "function" && define.amd) {',
                 '        // AMD. Register as an anonymous module with a dependency on Rangy.',
-                '        define(["./rangy-core"], factory);',
+                '        if ( require && require.s.contexts._.defined.rangy ) {',

```

Comment:
No way.

`require.s` is **private** to RequireJS. If you want to mess with it in your own code, that's on you. But making widely-used library mess with RequireJS' private parts is a non-starter.